README.md

```diff
@@ -10,11 +10,7 @@ For inspiration, see https://github.com/jakevdp/altair-transform
 pip install ibis-vega-transform
 jupyter labextension install \
     ibis-vega-transform \
-    "tornado<6" \
-    jupyterlab-server-proxy-saulshanabrook # optional, if you want to see icon in JL to launch tracing GUI
-
-# Optionally install `jaeger-all-in-one` binary and add to path to see tracing
-# https://www.jaegertracing.io/download/
+    "tornado<6"
```

Comment:
@saulshanabrook I don't understand this command?

You meant to write
```
pip install ibis-vega-transform "tornado<6"
jupyter labextension install ibis-vega-transform
```

?

Also, should not we update the `setup.py` to include the tornado constraint?

Reply:
Yep and yep! Just opened https://github.com/Quansight/ibis-vega-transform/issues/32 to track the second